databricks/koalas/series.py

```diff
@@ -89,6 +90,7 @@ class Series(_Frame):
     :ivar _index_info: Each pair holds the index field name which exists in Spark fields,
       and the index name.
     """
+    plot = CachedAccessor("plot", KoalasSeriesPlotMethods)
```

Comment:
is this our lazy_property defined in utils.py?

Reply:
Yes, it was not there when I started my PR, but I've merged master into my branch and changed code to use it.